integration-cli/requirements.go

```diff
@@ -88,3 +96,14 @@ func testRequires(t *testing.T, requirements ...TestRequirement) {
 		}
 	}
 }
+
+// testRequires checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func testRequiresWithLog(t *testing.T, msg string, requirements ...TestRequirement) {
```

Comment:
@LK4D4 more discussion about this is here: https://github.com/docker/docker/pull/11048#discussion_r26081777